index.js

```diff
@@ -260,11 +284,17 @@ module.exports = function (schema, options) {
             this.deletedId = params.deletedId
         }
 
-        if (options.validateBeforeDelete === false) {
-            return this.save({ validateBeforeSave: false }, cb);
+        const options = {};
+        if (params.session) {
+            options.session = params.session;
         }
```

Comment:
🤏 I'd do this in a similar way to what was done before
```suggestion
        var saveOptions = {};
        if (params.session) {
            saveOptions.session = params.session;
        }
```

Reply:
Also for this PR lets keep using `var`. There's more chance a PR back to the original project would be accepted that way.